src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -16,6 +16,10 @@ import ProgressButton from '@department-of-veterans-affairs/formation-react/Prog
 
 import { toggleLoginModal } from '../../site-wide/user-nav/actions';
 
+const DEFAULT_FORBIDDEN_MESSAGE = `
+  We're sorry. We can't give you access to this information. For help, please call the VA.gov help desk at 855-574-7286 (TTY: 711). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. ET.
```

Comment:
This needs content approval. @peggygannon

Reply:
Fortunately, she wrote it for us. :)